src/CodeGenerator/src/Definition/Operation.php

```diff
@@ -35,14 +35,17 @@ class Operation
     private $example;
 
     /**
-     * @var \Closure
+     * @var \Closure(string, Member|null=, array<string, mixed>=): Shape
      */
     private $shapeLocator;
 
     private function __construct()
     {
     }
 
+    /**
+     * @param \Closure(string, Member|null=, array<string, mixed>=): Shape $shapeLocator
```

Comment:
What does the tailing equals do?

Reply:
it marks the argument as optional in the callable signature (without it, you cannot omit the parameter when calling the callable).